content/en/blog/_posts/2022-12-16-statefulset-migration.md

```diff
@@ -0,0 +1,212 @@
+fully connected world, planned downtime and unavailability may not allow you to
+meet your business goals. You could use
+[Cascading Delete](/docs/tutorials/stateful-application/basic-stateful-set/#cascading-delete)
+or
+[On Delete](/docs/tutorials/stateful-application/basic-stateful-set/#on-delete)
+to migrate individual pods, however this is error prone and tedious to manage.
+You lose the self-healing benefit of the StatefulSet controller when your Pods
+fail or are evicted.
+
+This feature enables a StatefulSet to be responsible for a range of ordinals
+within a logical range of `[0, N)`. With it, you can scale down a range
+(`[0, k)`) in a source cluster, and scale up the complementary range (`[k, N)`)
+in a destination cluster, while maintaining application availability. This
+enables you to retain *at most one* semantics and
+[Rolling Update](/docs/tutorials/stateful-application/basic-stateful-set/#rolling-update)
+behavior when orchestrating a migration across clusters.
+
+## Why would I want to use this feature?
+
+Say you're running your StatefulSet in one cluster, and need to migrate it out
+to a different cluster. There are many reasons why you would need to do this:
+ * **Scalability**: Your StatefulSet has scaled too large for your cluster, and
+   has started to disrupt the quality of service for other workloads in your
+   cluster.
+ * **Isolation**: You're running a StatefulSet in a cluster that is accessed 
+   by multiple users, and namespace isolation isn't sufficient.
+ * **Cluster Configuration**: You want to move your StatefulSet to a different
+   cluster to use some environment that is not available on your current
+   cluster.
+ * **Control Plane Upgrades**: You want to move your StatefulSet to a cluster
+   running an upgraded control plane, and can't handle the risk or downtime of
+   in-place control plane upgrades.
+
+## How do I use it?
+
+Enable the `StatefulSetStartOrdinal` feature gate on a cluster, and create a
+StatefulSet with a customized `.spec.ordinals.start`.
+
+## Try it for yourself
+
+In this demo, you'll use the `StatefulSetStartOrdinal` feature to migrate a
+StatefulSet from one cluster to another. For this demo, the
+[redis-cluster](https://github.com/bitnami/charts/tree/main/bitnami/redis-cluster)
+Bitnami Helm chart is used to install Redis.
+
+Tools Required:
+ * [yq](https://github.com/mikefarah/yq)
+ * [helm](https://helm.sh/docs/helm/helm_install/)
+
+Pre-requisites: Two clusters named `source` and `destination`.
+ * `StatefulSetStartOrdinal` feature gate is enabled on both clusters
+ * [MultiClusterServices](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
+support is enabled
+ * The same default `StorageClass` is installed on both clusters. This
+   `StorageClass` should provision underlying storage that is accessible from
+   both clusters
+
+1. Create a demo namespace on both clusters.
```

Comment:
If using an ordered list (this _is_ an ordered list according to CommonMark), then indent the text up to the start of the next list item, so that it matches the first character of the text.

Avoid 4-character indents because in Markdown a 4 character indent can also mean monospace block.

Reply:
Thanks, updated these to 3-indent to match the first character of the text in the ordered list.